docs/spec/TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+< SPEC NAME >
+============================
+
```

Comment:
Perhaps a Motivation Section might also be prudent :)

Reply:
Oh I see... Motivation would go into the 'Specification' section... copy pasted from the doc :)